express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+let id = 1;
+const grades = [];
+
+app.use(express.json())
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
```

Comment:
This should send back an array of `grade` objects. You need to use a `for in` loop to build the array and then send it back to the client.

Reply:
![Screen Shot 2022-09-29 at 6 00 42 PM](https://user-images.githubusercontent.com/44100814/193167735-a881b9aa-863f-4263-8f3f-e1beeccc77e0.png)